khmer/_khmer.cc

```diff
@@ -4329,6 +4332,63 @@ labelhash_n_labels(khmer_KGraphLabels_Object * me, PyObject * args)
 
 static
 PyObject *
+labelhash_label_across_high_degree_nodes(khmer_KGraphLabels_Object * me,
+        PyObject * args)
+{
+    LabelHash * labelhash = me->labelhash;
+
+    const char * long_str;
+    khmer_HashSet_Object * hdn_o = NULL;
+    Label label;
```

Comment:
What is a label? Type wise it is an `uint64` which makes me think that in the future it also needs to be a `BigHashType`. But I don't actually really know what the concept of a label is (from a science POV).